writing/developmentdocs/migration_template.md

```diff
@@ -0,0 +1,214 @@
+# Migrating categories and sub categories in a dynamodb table
+
+This document aims to map the categories and subcategories of articles present in dynamodb table with new categories and sub categories (whenever necessary) .
+
+## 0. Prerequisite knowledge/work done
+
+1. Each table stores what kind of data and what interface is used to store the article .
+2. Knowledge about how to create APIs using NodeJs and how do they work .
+
+## 1. Prior Understanding
+
+Some important points :
+
+1. when to use different dynamodb commands like getcommand,scancommand,putcommand .what is event object and how to use it .
+2. how to write parameters object to pass in these different commands.
+3. how to use get,put,get,post command from aws-amplify/api library and how to pass query parameters.
+
+## API permissions
+
+currently the APi nbnwreactfrontend does not have permission to update articles in NBNWNewsTable so use the nbnweditorfinal api to update the items of this particular table.
+
+**Functions used in backend**
+<br />
+Steps involved
+<br />
+1 Use these functions defined here in your api change the tablename,filterexpression,expressionattributevalues depending on table attributes.
+<br />
+2 Run this command to update backend in cloud
+

```

Comment:
Please add steps one by one on how to perform. Assume that Tisha, with no prior experience will be performing these steps.